source/create-agg-pipeline.txt

```diff
@@ -1,29 +1,43 @@
 .. _compass-agg-builder:
+.. _create-agg-pipeline:
 
-============================
-Aggregation Pipeline Builder
-============================
+==============================
+Create an Aggregation Pipeline
+==============================
 
 .. default-domain:: mongodb
 
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-*New in version 1.14.0.*
-
 The Aggregation Pipeline Builder in |compass| lets you create
 :manual:`aggregation pipelines </core/aggregation-pipeline/>` to process
-documents from a collection or view and return computed results. For
-example, you can use aggregation pipelines to:
+documents from a collection or view and return computed results.
+
+About This Task 
+---------------
+
+|compass| provides different modes to create aggregation pipelines:
+
+- Stage View Mode, a visual pipeline editor that preloads pipeline syntax based 
+  on your selected stages.
 
-- Group values from multiple documents together.
+- Stage Wizard, a feature of Stage View Mode that provides a set of templates 
+  for simple aggregation stage use cases. The Stage Wizard only includes simple 
+  use cases to help you get started with your aggregation pipeline.
 
-- Perform operations on the grouped data to return a single result.
+- Focus Mode, a feature of Stage View Mode where you edit one pipeline stage at 
```

Comment:
[praise, question] Nice work with these updated descriptions! Would it make sense to nest Stage Wizard and Focus Mode under Stage View Mode?

Reply:
Thanks! I just took a look at what the staging might look like with Stage Wizard and Focus Mode nested, but given they do stand as different, individual modes for creating an aggregation pipeline, I'm leaning towards keeping as is. Otherwise, I feel like users might expect the features to be integrated within the Stage View Mode procedures and/or the UI.